### PR TITLE
create event list and link to event pages. extract to components

### LINKED ISSUE #222: Campaign page event list
The campaign page event list should contain:

* Event title (or activity title if `!title`)
* Start and end date/time
* Location

The item should link to the event page.

### PR DESCRIPTION
**Pull request by nehahirve**
This PR
Creates a reusable event list component that links each event to the correct page and displays relevant info
Resolves #222
Screenshot 2021-07-05 at 16 13 18

### CODE REVIEW

File: src/pages/organize/[orgId]/campaigns/[campId]/index.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 import { GetServerSideProps } from 'next';
 import { grey } from '@material-ui/core/colors';
-import { FormattedMessage as Msg } from 'react-intl';
 import NextLink from 'next/link';
-import { Avatar, Box, Button, Link, List, ListItem, makeStyles, Typography } from '@material-ui/core';
+import { Avatar, Box, Button, Link, makeStyles, Typography } from '@material-ui/core';
+import { FormattedDate, FormattedMessage as Msg } from 'react-intl';
 import { Phone, PlaylistAddCheck, Public, Settings } from '@material-ui/icons';
 
+import EventList from '../../../../../components/organize/Event/EventList';
```
Comment (richardolsson) [inline src/pages/organize/[orgId]/campaigns/[campId]/index.tsx:8]: You have put this component in a directory called "Event". At first glance that looks odd to me. What is the reasoning behind it?

Comment (richardolsson) [inline src/pages/organize/[orgId]/campaigns/[campId]/index.tsx:8]: You have put this component in a directory called "Event". At first glance that looks odd to me. What is the reasoning behind it?

**Review by richardolsson — CHANGES_REQUESTED** [review]
Looks great! Only thing I'm reacting to is the path of directories. The "Event" directory seems off to me, both in purpose and naming.

Regarding purpose: Is the purpose of this directory to contain all components that are related to events? That might make sense, I don't know, but are we ready to make that decision? Last time we discussed this, I think the conclusion was to just keep things like they were for some more time before deciding on a directory scheme, but I'm ready to discuss!

Regarding capitalization: The capitalization of the `EventList` directory makes sense to me, because it's code is in the `index.tsx` file which means it can be imported as `src/path/to/EventList`. The `EventList` directory is capitalized as if it was a component module, because it can be used as if it was a component module.

The `Event` directory however is just a directory, like `components` or `layout`. It does not contain an `index.tsx` and will never be imported. So if we want to keep the directory, I think it should be renamed `event`.

But I'm leaning towards just removing the `Event` directory and moving it's contents up one level while we ponder The Perfect Structure™. 😊

**Comment by nehahirve** [comment]
moved one directory up now. Kept it in the /organize directory so it doesn't clash with another, older, component named EventList. The other option was to name the component OrganizeEventList and move it 2 directories up, but thought that the naming was getting too convoluted in that case

**Review by richardolsson — APPROVED** [review]
Great! 🙌

**Pull request merged**